tests/Doctrine/Tests/Common/Annotations/SimpleAnnotationReaderTest.php

```diff
@@ -9,6 +9,8 @@ class SimpleAnnotationReaderTest extends AbstractReaderTest
     /**
      * Contrary to the behavior of the default annotation reader, we do just ignore
      * these in the simple annotation reader (so, no expected exception here).
+     *
+     * @doesNotPerformAssertions
```

Comment:
Tests that do not assert anything sound pretty useless to me.

Reply:
We've talked at length about these. The original test checks to ensure the annotation read throws an exception. The simple reader doesn't throw an exception, which is why this test overrides the original but doesn't make an assertion (the absence of an exception is all we need to assert). Hence the annotation.